treenode_test.go

```diff
@@ -194,3 +210,55 @@ func (s *spawnProto) HandlerError2(msg spawnMsg) error {
 func (s *spawnProto) HandlerError3(msg spawnMsg) (int, error) {
 	return 0, nil
 }
+
+// Simple protocol to have messages transmit between nodes
+const dummyName = "TestDummy"
```

Comment:
can you call this `pingPongProto` or something like that, i think that's what it does